docker-compose.yml

```diff
@@ -27,6 +27,7 @@ services:
       DB_USERNAME: ${DB_USERNAME:-atlas}
       DB_PASSWORD: ${DB_PASSWORD:-iamatlas}
       SESSION_SECRET: ${SESSION_SECRET:-supersecret}
+      GOOGLE_MAPS_JS_API_KEY: ${GOOGLE_MAPS_JS_API_KEY:-GOOGLE_MAPS_JS_API_KEY}
```

Comment:
I'm not sure what are you trying to do here. I understand the comment, also it would be ideal to add a link on how to generate the google maps js api.

Reply:
@cintiadr I'm sort of new to docker but "I tried to create an environment variable called GOOGLE_MAPS_JS_API_KEY". e_e

I see that the docker-compose.yml has been updated with the change, so I omitted it out of my PR.